README.md

```diff
@@ -1,6 +1,6 @@
 # XRegExp 4.4.0
 
-[![Build Status](https://travis-ci.org/slevithan/xregexp.svg?branch=master)](https://travis-ci.org/slevithan/xregexp)
+[![Build Status](https://github.com/slevithan/xregexp/workflows/.github/workflows/build.yml/badge.svg)](https://github.com/slevithan/xregexp/actions)
```

Comment:
nice

Reply:
Thanks, I _think_ it should work, but we can check once it's merged.

Reply:
fixed in https://github.com/slevithan/xregexp/pull/310